config/izum_vega.py

```diff
@@ -0,0 +1,119 @@
+from os import environ
+username = environ.get('USER')
+
+# This is an example configuration file
+site_configuration = {
+    'general': [
+        {
+            'remote_detect': True,
+        }
+    ],
+    'systems': [
+        {
+            'name': 'vega',
+            'descr': 'Vega, a EuroHPC JU system',
+            'modules_system': 'lmod',
+            'hostnames': ['vglogin*','cn*','gn*'],
+            'stagedir': f'reframe_runs/staging',
+            'outputdir': f'reframe_runs/output',
+            'partitions': [
+                {
+                    'name': 'cpu',
+                    'scheduler': 'slurm',
+                    'prepare_cmds': [
+                        'source /cvmfs/pilot.eessi-hpc.org/latest/init/bash',
+                        'export SLURM_EXPORT_ENV=ALL',
+                        # Avoid https://github.com/EESSI/software-layer/issues/136
+                        # Can be taken out once we don't care about old OpenMPI versions anymore (pre-4.1.1)
+                        'export OMPI_MCA_pml=ucx',
+                    ],
+                    'launcher': 'mpirun',
+                    'access':  ['-p cpu', '--export=None'],
+                    'environs': ['default'],
+                    'max_jobs': 120,
+                    'features': [
+                        'cpu',
+                    ],
+                    'descr': 'CPU partition Standard, see https://en-doc.vega.izum.si/architecture/'
+                },
+                {
+                    'name': 'gpu',
+                    'scheduler': 'slurm',
+                    'prepare_cmds': [
+                        'source /cvmfs/pilot.eessi-hpc.org/latest/init/bash',
+                        'export SLURM_EXPORT_ENV=ALL',
+                        # Avoid https://github.com/EESSI/software-layer/issues/136
+                        # Can be taken out once we don't care about old OpenMPI versions anymore (pre-4.1.1)
+                        'export OMPI_MCA_pml=ucx',
+                    ],
+                    'launcher': 'mpirun',
+                    'access':  ['-p gpu', '--export=None'],
+                    'environs': ['default'],
+                    'max_jobs': 60,
+                    'devices': [
+                        {
+                            'type': 'gpu',
+                            'num_devices': 4,
+                        }
+                    ],
+                    'resources': [
+                        {
+                            'name': '_rfm_gpu',
+                            'options': ['--gpus-per-node={num_gpus_per_node}'],
+                        }
+                    ],
+                    'features': [
+                        'gpu',
```

Comment:
also add extras to specify `GPU_VENDOR` (+ use constants)

Reply:
Done, though it will have to be updated once https://github.com/EESSI/test-suite/pull/60/files is merged, because that includes further definitions of constant (`GPU` is also a constant there...)